grails-app/services/org/pih/warehouse/inventory/ProductAvailabilityService.groovy

```diff
@@ -166,8 +166,24 @@ class ProductAvailabilityService {
             }
             picklistItems {
                 if (product) {
-                    requisitionItem {
-                        eq("product", product)
+                    or {
+                        requisitionItem {
+                            eq("product", product)
+                        }
+                        orderItem {
+                            eq("product", product)
+                        }
+                    }
+                }
+                requisitionItem {
```

Comment:
this will remove all returns, because for those lines requisition item is null
I think it should be order item not null or (requisition item not null and not recalled and not on hold)

Reply:
true